Makefile

```diff
@@ -105,17 +105,20 @@ $(BUILD_DIR)/image/partitions.csv: $(TOIT_PATH)/toolchains/esp32/partitions.csv
 $(BUILD_DIR)/image/system.snapshot: $(TOIT_PATH)/build/esp32/ $(BUILD_DIR)/image/
 	cp $(TOIT_PATH)/build/esp32/system.snapshot $@
 
-$(BUILD_DIR)/image/jaguar.snapshot: $(TOIT_PATH)/build/esp32/ $(BUILD_DIR)/image/
-	cp $(TOIT_PATH)/build/esp32/program.snapshot $@
+$(BUILD_DIR)/image/jaguar.snapshot: install-dependencies
+$(BUILD_DIR)/image/jaguar.snapshot: $(JAG_TOIT_PATH)/bin/toit.compile $(BUILD_DIR)/image/
+	$(JAG_TOIT_PATH)/bin/toit.compile -w $@ $(JAG_ENTRY_POINT)
 
 .PHONY: image
+image: $(BUILD_DIR)/image/jaguar.snapshot
+image: $(BUILD_DIR)/image/firmware.envelope
+# TODO(kasper): The remaining stuff should be bundled in the
+# firmware.envelope file.
 image: $(BUILD_DIR)/image/toit.elf
-image: $(BUILD_DIR)/image/toit.bin
 image: $(BUILD_DIR)/image/bootloader/bootloader.bin
 image: $(BUILD_DIR)/image/partitions.bin
 image: $(BUILD_DIR)/image/partitions.csv
 image: $(BUILD_DIR)/image/system.snapshot
-image: $(BUILD_DIR)/image/jaguar.snapshot
 
 .PHONY: install-esp-idf
 install-esp-idf:
```

Comment:
I think you want to remove `--auto-sync` from the `install-dependencies`. Otherwise the package manager won't find all packages.

Reply:
Done.